src/main/docker/Dockerfile

```diff
@@ -60,6 +60,7 @@ RUN zypper -n refresh && \
     zypper -n update && \
     zypper -n install curl postgresql dejavu-fonts crypto-policies-scripts && \
     update-crypto-policies --set DEFAULT:DISABLE-CBC && \
+    sed -rie '/^jdk.tls.disabledAlgorithms=/s/$/, TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256/' /etc/crypto-policies/back-ends/java.config && \
```

Comment:
Don't go over the 138-character guideline.  Use a line continuation character to split this across two lines.